LinkedLists/AddTwoNumbers.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-class MainClass {
+class AddTwoNumbers {
```

Comment:
Write a couple of lines that describe what this class is doing. preferably by showing an example